src/tribler-core/tribler_core/modules/libtorrent/download.py

```diff
@@ -5,7 +5,7 @@
 """
 import base64
 import logging
-from asyncio import CancelledError, Future, TimeoutError, iscoroutine, sleep, wait_for
+from asyncio import CancelledError, Future, iscoroutine, sleep, wait_for
```

Comment:
The `wait_for` function throws an `asyncio.TimeoutError`. Please put it back and optionally rename it like you did elsewhere.

Reply:
ok, that seems to be missed out.